bugsnag-android-core/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -250,7 +244,7 @@ public static void notify(@NonNull final Throwable exception,
     public static void notify(@NonNull String name,
                               @NonNull String message,
                               @NonNull StackTraceElement[] stacktrace) {
```

Comment:
Yep, this is an intentional deviation that I believe makes sense for Android alone. Libraries such as `LeakCanary` have a bugsnag integration that captures a trace of memory leaks in Android applications, which makes more sense to capture as a `StacktraceElement[]` rather than an exception object. Without the `StacktraceElement[]` I don't think it'd make much sense to include this method.

Reply:
That might be a good reason to have this as an additional override for Android, but I think we should still conform to the spec by having a name/message[/onError] override so that developers can easily call `notify` without any error constructs.

Reply:
The latest version of the spec states that it's up to each individual notifier on to whether it makes sense to implement the name/message/onerror overload. I don't think there would be much value in allowing users to send events without a stacktrace on Android - but maybe I'm missing a usecase